src/django_components/slots.py

```diff
@@ -67,29 +67,6 @@ def __call__(self, ctx: Context, slot_data: TSlotData, slot_ref: "SlotRef") -> S
 # Internal type aliases
 SlotId = str
 SlotName = str
-SlotDefaultName = str
-SlotDataName = str
-
-
-@dataclass(frozen=True)
-class FillContent(Generic[TSlotData]):
```

Comment:
No longer needed, the `slot_default_var` and `slot_data_var` are now part of the slot function.